lunabot_bt:
  beac_UUID1: 6077718e7625,
  beac_UUID2: 6077718e6b7b,
  beac_UUID3: 6077718e70af,
  beac_UUID4: 6077718e6b2f